tests/gtfs_reader.rs

```diff
@@ -0,0 +1,35 @@
+extern crate navitia_model;
+extern crate tempdir;
+
+use navitia_model::gtfs;
+use tempdir::TempDir;
+use std::fs::File;
+use std::io::prelude::*;
+
+#[test]
+fn load_minimal_agency() {
+    let agency_content = "agency_name,agency_url,agency_timezone\n
+    My agency,http://my-agency_url.com,Europe/London";
+    let tmp_dir = TempDir::new("osm_transit_extractor").expect("create temp dir");
+    let file_path = tmp_dir.path().join("agency.txt");
+    let mut f = File::create(&file_path).unwrap();
+    f.write_all(agency_content.as_bytes()).unwrap();
```

Comment:
better to add the files in the repo and read them.

Reply:
I started with a fixture, but it led to an awful list of subdirectories. I thought it best to create the file one the fly for those simple tests.